app/javascript/components/dashboard/configuration-profiles/edit/EditConfigurationProfile.jsx

```diff
@@ -116,6 +116,16 @@ const EditConfigurationProfile = (props) => {
 const CPCardHeader = () => {
   const configurationProfile = useSelector((state) => state.currentCP);
   const savingCP = useSelector((state) => state.savingCP);
+  const [showChangesSaved, setShowChangesSaved] = useState(false);
+
+  useEffect(() => {
+    if (savingCP) {
+      setShowChangesSaved(true);
+      setTimeout(() => {
```

Comment:
The timeout needs to be cleared during the cleanup phase in order to avoid a state update if the component gets unmounted.

Reply:
@excelsior corrected.